apps/nestjs-backend/src/db-provider/filter-query/filter-query.abstract.ts

```diff
@@ -182,6 +182,7 @@ export abstract class AbstractFilterQuery implements IFilterQueryInterface {
     return (
       value !== null ||
       field.type === FieldType.Checkbox ||
+      (field.type === FieldType.Formula && field.cellValueType === CellValueType.Boolean) ||
```

Comment:
why not simplify just `field.cellValueType === CellValueType.Boolean`

Reply:
what will happen when field.isMultipleCellvalue = true

Reply:
When isMultipleCellValue is true, it will perform the filtering operation for the multiple boolean values